apps/web/package.json

```diff
@@ -21,26 +21,33 @@
   "dependencies": {
     "@kiwicom/margarita-core": "^0",
     "@kiwicom/margarita-relay": "^0",
+    "@kiwicom/margarita-utils": "^0",
+    "@kiwicom/margarita-components": "^0",
     "file-loader": "^3.0.1",
     "next": "^7.0.2",
     "next-images": "^1.0.4",
     "next-plugin-transpile-modules": "^0.1.3",
     "react-native-web": "^0.10.0",
     "schedule": "^0.4.0",
-    "url-search-params-polyfill": "^5.0.0"
+    "url-search-params-polyfill": "^5.0.0",
+    "webpack-bundle-analyzer": "^3.0.4"
```

Comment:
Does this one needs to be in `dependencies` and not in `devDependencies`?

Reply:
It will generate an eslint-error if it is not, so yes

Reply:
Oh, I see. Thanks
So approved 🚀